src/tests/AvoInspector.test.ts

```diff
@@ -142,60 +153,102 @@ describe("Initialization", () => {
     expect(inspector.environment).toBe(AvoInspectorEnv.Staging);
   });
 
-  test("Version", () => {
+  test("Prod env is set using AvoInspectorEnv", () => {
+    // When
+    let inspector = new AvoInspector({
+      apiKey: "api-key-xxx",
+      env: AvoInspectorEnv.Prod,
+      version: "0",
+    });
+
+    // Then
+    expect(inspector.environment).toBe(AvoInspectorEnv.Prod);
+  });
+
+  test("Prod environment is set using string", () => {
+    // When
+    let inspector = new AvoInspector({
+      apiKey: "api-key-xxx",
+      env: "prod",
+      version: "0",
+    });
+
+    // Then
+    expect(inspector.environment).toBe(AvoInspectorEnv.Prod);
+  });
+
+  // TODO: is other environmets supported
```

Comment:
Should be 3 environments - dev, prod and staging

Reply:
Maybe then should fall back to dev environment in case other than dev, prod, staging environment is used? Same as it happening when no environment is provided?

Reply:
Should I add this functionality/check or you will add that?

Reply:
Would be great if you could take it